### PR TITLE
feat:azure webcloning:completed header part of mobile and tab

### PR DESCRIPTION
**Pull request by theresmaryjohn**

### CODE REVIEW

File: Marioclubtask11/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (APJ-9) [inline Marioclubtask11/.vscode/settings.json:2]: Add this file to .gitignore

File: task-13/azure/index.html [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+    <div class="header">
+        <div class="nav-hamburger"><img src="img/svgexport-1.svg" alt=""><strong>Azure</strong></div>
+        <nav class="mic-icon"><img src="img/svgexport-3.svg" alt=""></nav>
+        <div>
+            <nav id="find"><img src="img/icons8-search-25.png"></nav>
+            <nav id="who"><a href="">Sign In</a></nav>
+        </div>
+
+    </div>
+    <div class="header-1">
```
Comment (APJ-9) [inline task-13/azure/index.html:26]: make class names meaningful

File: task-13/azure/style.css [hunk 3]
```diff
@@ -0,0 +1,2439 @@
+body, a, strong, li{
+    list-style-type: none;
+    text-decoration: none;
+    margin:0;
+    padding: 0;
+    
+}
+*{
```
Comment (APJ-9) [inline task-13/azure/style.css:8]: root must be in top

File: task-13/azure/style.css [hunk 4]
```diff
@@ -0,0 +1,2439 @@
...
+    width:68px;
+    height: 68px;
+}
+
+
+div.easy-m strong{
+ font-size: 20px;
+ font-family: 'Segoe UI';
+ color: #0062ad;
+ margin-block: 19px;
```
Comment (APJ-9) [inline task-13/azure/style.css:274]: format the code and follow only any one convention

File: task-13/azure/style.css [hunk 5]
```diff
@@ -0,0 +1,2439 @@
...
+    font-size: 14px;
+    font-family: 'Segoe UI';
+    font-weight: 600;
+}
+.feat-m4{
+    align-items: center;
+    box-shadow: 0 2px 5px rgb(0 0 0 / 14%);
+    border: 1px solid transparent;
+    background-color: white;
+    border-radius: 5px;
```
Comment (APJ-9) [inline task-13/azure/style.css:451]: group similar styles

**Comment by theresmaryjohn** [comment]
file:///home/theres/Downloads/127.0.0.1_5500_index.html%20(3).png
file:///home/theres/Downloads/127.0.0.1_5500_index.html.png
file:///home/theres/Downloads/127.0.0.1_5500_index.html%20(1).png

File: task-13/azure/style.css [hunk 6]
```diff
@@ -0,0 +1,2152 @@
...
+    transform: translateY(-5px);
+}
+.feat-tab-find:hover{
+    box-shadow: 0px 10px 20px 5px rgba(138, 129, 129, 0.5);
+    transform: translateY(-5px);
+}
+.logo-tab .logo-t2:hover{
+    filter: grayscale(0%);
+}
+.logo-tab .logo-t3:hover{
```
Comment (APJ-9) [inline task-13/azure/style.css:58]: can we group similar styles?

`
.logo-tab .logo-gray:hover{
    filter:grayscale(0%);
}
`

File: task-13/azure/style.css [hunk 7]
```diff
@@ -0,0 +1,2152 @@
...
+    font-family: 'Segoe UI';
+    font-weight: 20px;
+    background-color: transparent;
+ }
+ .mobile-main{
+    padding: 0;
+    margin: 0;
+    overflow: hidden;
+ }
+ .mainpart{
```
Comment (APJ-9) [inline task-13/azure/style.css:148]: try to give meaningful class names

File: task-13/azure/style.css [hunk 8]
```diff
@@ -0,0 +1,2152 @@
...
+    font-weight: 600;
+}
+.features-m{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-block-start: -11px;
+    width: 95%;
+}
+.features-mob-find{
```
Comment (APJ-9) [inline task-13/azure/style.css:416]: try to group similar styles together 
`
{
display styles;
display height position ........
text styles;
font, color text-align......
}

`

File: task-13/azure/style.css [hunk 9]
```diff
@@ -0,0 +1,2152 @@
...
+    align-items: center;
+    box-shadow: 0 2px 5px rgb(0 0 0 / 14%);
+    background-color: white;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    text-align: center;
+    width: 101%;
```
Comment (APJ-9) [inline task-13/azure/style.css:425]: why 101% here ?

File: task-13/azure/style.css [hunk 10]
```diff
@@ -0,0 +1,2152 @@
...
+    grid-row: 1/span2;
+    background: url(img/desktop1icon.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+   border-radius: 2px;
+   border: 1px solid transparent;
+   box-shadow: 0 2px 5px rgb(0 0 0 / 14%);
+   margin-inline-start: 14px;
+   display: flex;
+   align-items: center;
```
Comment (APJ-9) [inline task-13/azure/style.css:1153]: format the code always

File: task-13/azure/style.css [hunk 11]
```diff
@@ -0,0 +1,2152 @@
...
+    padding-left: 19px;
+    padding-bottom: 70px;
+}
+.feat-tab{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    grid-auto-rows: 1fr;
+    margin: 0;
+    padding: 0;
```
Comment (APJ-9) [inline task-13/azure/style.css:1284]: why padding 0 here?

File: task-13/azure/style.css [hunk 12]
```diff
@@ -0,0 +1,2152 @@
...
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
+    }
+    .place-mob p{
+        margin-top: -7px;
+    }
+    .explore-mob{
+        display: grid;
+        column-gap: 7px;
+        grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
```
Comment (APJ-9) [inline task-13/azure/style.css:1868]: can we use shorthand notation here?

File: task-13/azure/style.css [hunk 13]
```diff
@@ -0,0 +1,2152 @@
...
+    justify-content: center;
+}
+.microsoft-icon-tab li a{
+    color: black;
+    font-weight: 500;
+    font-family:'Segoe UI',Segoe UI,'Helvetica Neue',Helvetica,Arial, sans-serif  ;
+    letter-spacing: 0.4px;
+}
+.microsoft-icon-tab img{
+margin: 3px;
```
Comment (APJ-9) [inline task-13/azure/style.css:1935]: why margin 3 here?

File: task-13/azure/style.css [hunk 14]
```diff
@@ -0,0 +1,2484 @@
...
+}
+
+.feat-tab-find:hover {
+    box-shadow: 0px 10px 20px 5px rgba(138, 129, 129, 0.5);
+    transform: translateY(-5px);
+}
+
+.logo-tab .logo-t2:hover,
+.logo-tab .logo-t3:hover,
+.logo-tab .logo-t4:hover,
```
Comment (APJ-9) [inline task-13/azure/style.css:71]: can we use the same class names for all these sections?

File: task-13/azure/style.css [hunk 15]
```diff
@@ -0,0 +1,2484 @@
...
+        margin: 0;
+        padding: 0;
+        background: url(img/december-hero-mobile.png);
+        background-repeat: no-repeat;
+        background-size: cover;
+        background-position: center;
+        width: 100%;
+        height: fit-content;
+        padding-bottom: 69px;
+
```
Comment (APJ-9) [inline task-13/azure/style.css:175]: remove the unwanted space

**Comment by APJ-9** [comment]
Add Screenshots

**Pull request merged**